src/OpenTelemetry.Api/Trace/SpanAttributeConstants.cs

```diff
@@ -40,9 +40,10 @@ public static class SpanAttributeConstants
         public const string HttpRouteKey = "http.route";
         public const string HttpFlavorKey = "http.flavor";
 
-        public const string DatabaseTypeKey = "db.type";
-        public const string DatabaseInstanceKey = "db.instance";
+        public const string DatabaseSystemKey = "db.system";
+        public const string DatabaseNameKey = "db.name";
         public const string DatabaseStatementKey = "db.statement";
+        public const string DatabaseStatementTypeKey = "db.statement_type";
```

Comment:
Is this part of spec?

Reply:
Oh I now recollect we already discussed this. Will need to follow up on spec. Note to myself!